### PR TITLE
treewide: remove rob

### PR DESCRIPTION
**Pull request by AndersonTorres**
Since theey is inactive.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/libraries/opentracing-cpp/default.nix [hunk 1]
```diff
@@ -15,7 +21,6 @@ stdenv.mkDerivation rec {
     description = "C++ implementation of the OpenTracing API";
     homepage = "https://opentracing.io";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ rob ];
+    maintainers = [ ];
```
Comment (AndersonTorres) [inline pkgs/development/libraries/opentracing-cpp/default.nix:24]: This package should  be deleted.

**Review by SuperSandro2000 — COMMENTED** [review]
I am not a fan that the same commit also did a bunch of reformatting. Also why weren't they also remove from the maintainers list?

File: pkgs/development/libraries/opentracing-cpp/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/libraries/opentracing-cpp/default.nix:24]: It is a dependency of a nginx module so we cannot just delete it.

**Comment by AndersonTorres** [comment]
> I am not a fan that the same commit also did a bunch of reformatting.

The formatting is required. But since there are too few packages here, I will split  the things.

> Also why weren't they also remove from the maintainers list?

I feel this is not needed. Also it is a bit problematic for packages maintained by multiple silently-retired maintainers. It is better to do this in another round.

**Comment by bendlas** [comment]
Closing these removal PRs, since they are not grounded within community consensus. See https://github.com/NixOS/nixpkgs/pull/337478#issuecomment-2310779183

Don't know the GH username of rob, but be sure that we value your involvement and any time you may have to spare, helping with the packages you're maintaining and others as well.

**Comment by SuperSandro2000** [comment]
it is @rbvermaa and that account is super inactive, so we are just pretending that the packages are maintained tbh

**Pull request closed without merging**